### ISSUE TITLE
Hyperparameter suggestion for finetuning on new smaller dataset 

### ISSUE THREAD

**Issue by sahithyaravi** [Zejun-Yang/AniPortrait#199]
Hello,

I tried using your stage2 training code to finetune on a new set of videos from [MEAD](https://github.com/uniBruce/Mead). However, after finetuning for ~1000 steps with a learning rate of 1e-6, I observed that the loss does not really drop.

Do you have an idea on why this might happen? How many steps would you recommend for finetuning on a new set of videos?

Here is the link to the loss curve:
https://wandb.ai/sahithyaravi/anipotrait-stage2/reports/Stage-2-Finetune-on-MEAD-1000-steps---Vmlldzo5NjU0OTgx?accessToken=bc46dmtyi5dsbizyhk0ee5ltc1sr3yahl7d5xnvu3la87swjg2t3wnbiz3euyha9

Here are my hyperparams

```
data:
  json_path: "train.json"
  sample_size:  [512, 512]
  sample_stride: 1
  sample_n_frames: 16
  sample_stride_aug: True


solver:
  gradient_accumulation_steps: 8
  mixed_precision: 'fp16'
  enable_xformers_memory_efficient_attention: True 
  gradient_checkpointing: True 
  max_train_steps: 1000
  max_grad_norm: 1.0
  # lr
  learning_rate: 1e-6
  scale_lr: False 
  lr_warmup_steps: 1
  lr_scheduler: 'constant'
  # optimizer
  use_8bit_adam: True 
  adam_beta1: 0.9
  adam_beta2: 0.999
  adam_weight_decay:  1.0e-2
  adam_epsilon: 1.0e-8

val:
  validation_steps: 500
  validation_steps_tuple: [1, 10, 20, 50, 80, 100, 200, 300, 400, 500]

noise_scheduler_kwargs:
  num_train_timesteps: 1000
  beta_start:          0.00085
  beta_end:            0.012
  beta_schedule:       "linear"
  steps_offset:        1
  clip_sample:         false

```
